arranger/namespace.yml

```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: overture
```

Comment:
Adding a namespace is good, but then you need to add a "namespace: overture" metadata entry for all your resources.

Alternatively, for a slightly DRYer experience, you can add a "namespace" entry in your kustomization file and kustomize will take care of adding a "namespace" metadata field to all the resources for you: https://kubernetes-sigs.github.io/kustomize/api-reference/kustomization/namespace/